internal/k8s/controller.go

```diff
@@ -2378,13 +2396,25 @@ func (lbc *LoadBalancerController) getPolicies(policies []conf_v1.PolicyReferenc
 		}
 
 		policy := policyObj.(*conf_v1.Policy)
-
+		msg := ""
```

Comment:
Is it more conventional to use `var msg string` here instead when initialising a variable to its zero-value?
I'm fairly new to golang so I could be wrong.